client/app/components/Toast/Toast.scss

```diff
@@ -0,0 +1,22 @@
+@import "~styles/_global.scss";
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
```

Comment:
We don't use this font in our app, so this is unnecessary.

Reply:
Acknowledged.